docs/js/extra.js

```diff
@@ -23,7 +23,7 @@ function setCopyText() {
   const attr = "clipboardText";
   // all "copy" buttons whose target selector is a <code> element
   const elements = document.querySelectorAll(
-    'button.md-clipboard[data-clipboard-target$="code"]',
+    'button.md-clipboard[data-clipboard-target$="code"], button.md-code__button[data-clipboard-target$="code"]',
```

Comment:
Could also consider just use
```suggestion
 'button[data-clipboard-target$="code"]',
 ```

Reply:
Seems reasonable too.